medication statements/README.md

```diff
@@ -0,0 +1,363 @@
+    },
+    "effectivePeriod": {
+      "start": "2010-02-10",
+      "end": "2010-02-10"
+    },
+    "dateAsserted": "2010-02-10T08:31:19+00:00",
+    "subject": {
+      "reference": "Patient/c782c5ef-8235-4acf-945b-268a1bbe9d64"
+    },
+    "taken": "unk",
+    "dosage": [
+      {
+        "text": "To Be Used As Directed"
+      }
+    ]
+  }
+}
+```
+</details>
+
+## JSON FHIR > XML HL7
+
+A XML HL7 `MedicationStatement` is mapped from a FHIR `medicationRequest`.  This can be one of two types, `ORDER` or `PLAN` which is determined by value of `medicationRequest.intent`
+
+### Order
+
+For details of order mapping see the JSON FHIR > XML HL7 mapping for [Medication](../medications/README.md)
+
+### Plan
+
+| Mapped to (XML HL7)                                                                                                | Mapped from (JSON FHIR / other source )                                                                                            |
+|--------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|
+| MedicationStatement \[@classCode] \[@moodCode]                                                                     | fixed value `@classCode="SBADM"`, fixed value `@moodCode="INT"`                                                                    |
+| MedicationStatement / id \[@root]                                                                                  | adaptor generated UUID                                                                                                             |
+| MedicationStatement / statusCode                                                                                   | `medicationRequest.status` when `"active"` = `"ACTIVE"` otherwise is `"COMPLETE"`                                                  |
+| MedicationStatement / effectiveTime / low \[@value]                                                                | `medicationRequest.dispenseRequest.validityPeriod.start`                                                                           |
+| MedicationStatement / effectiveTime / high \[@value]                                                               | `medicationRequest.dispenseRequest.validityPeriod.end` <sup>1</sup>                                                                |
+| MedicationStatement / availabilityTime  \[@value]                                                                  | `medicationRequest.dispenseRequest.validityPeriod.start`                                                                           |
+| MedicationStatement / consumable \[@typeCode] / manufacturedProduct \[@typeCode="CSM"]                             | fixed value = `"CSM"`                                                                                                              |
```

Comment:
I find this confusing, `manufacturedProduct` doesn't have a typeCode from what I can see.

Reply:
yeah I think I've accidentally added an extra level on here by mistake